src/coverlet.collector/DataCollector/CoverletCoverageDataCollector.cs

```diff
@@ -0,0 +1,179 @@
+﻿namespace Coverlet.Collector.DataCollector
+{
+    using System;
```

Comment:
Please move `using` on top out of namespace

Reply:
Done